src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenRefReturnTests.vb

```diff
@@ -1123,6 +1123,10 @@ public class B
 End Module",
                 referencedCompilations:={comp1},
                 compilationOptions:=TestOptions.DebugExe)
+            ' This test failes on non english machines.
+            ' The expected output is generated by the runtime and is localized with the culture of the current machine.
+            ' Localized StringRessources of the runtime are not part of the Roslyn solution.
+            ' Setting DefaultThreadCurrentCulture to en-US does not help because CompileAndVerify starts an AppDomain.
```

Comment:
>Setting DefaultThreadCurrentCulture to en-US does not help because CompileAndVerify starts an AppDomain. [](start = 14, length = 104)

Modify code in the Main function above to change to invariant culture and restore at the end. The same way as it is done in Spilling_ExceptionInArrayAccess unit test for example.